### ISSUE TITLE
Issue when creating a layer NDFrame.convert_dtypes() got an unexpected keyword argument 'dtype_backend'

### ISSUE THREAD

**Issue by mfbenitezp** [developmentseed/lonboard#224]
When I create the layer in the example in the 30MapChallenge. and other examples, I could not make the part of creating a layer run, as I got this error, related to the utils.py 

```
TypeError                                 Traceback (most recent call last)
[<ipython-input-35-2cd075ed474a>](https://localhost:8080/#) in <cell line: 1>()
----> 1 layer = ScatterplotLayer.from_geopandas(
      2     gdf,
      3     get_fill_color=[255, 0, 0],
      4 )
      5 map_ = Map(layers=[layer])

1 frames
[/usr/local/lib/python3.10/dist-packages/lonboard/_utils.py](https://localhost:8080/#) in auto_downcast(df)
     40     # also have to manually cast to a non-geo data frame because it'll fail to convert
     41     # dtypes on a GeoDataFrame without a geom col
---> 42     casted_df = pd.DataFrame(df.select_dtypes(exclude="geometry")).convert_dtypes(
     43         infer_objects=True,
     44         convert_string=True,

TypeError: NDFrame.convert_dtypes() got an unexpected keyword argument 'dtype_backend'
```
One thing I've noticed is when I run `pip install lonboard ` I got the following error, at the end of the installation. 

```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
ibis-framework 6.2.0 requires pyarrow<13,>=2, but you have pyarrow 13.0.0 which is incompatible.
pandas-gbq 0.17.9 requires pyarrow<10.0dev,>=3.0.0, but you have pyarrow 13.0.0 which is incompatible.
```
This has happened when installing the library in Google Colab.

**Comment by kylebarron** [comment]
This means your pandas version is too old. It needs to be pandas >2.

For now, we can set a pandas 2.0 requirement, but in the long term, pandas and geopandas will be optional dependencies.

**Comment by mfbenitezp** [comment]
Thanks! @kylebarron for the fast reply. I was able to manage the issue, once I updated pandas. Based on what you said, May I suggest updating the notebooks would be enough, so people like me trying to replicate/practice in their own env or colab are aware of this condition. I can PR the notebook with that small suggestion if you like. and also another I've been working to plot 3M points from the car accidents in the UK, for one of my lectures.
